databuilder/databuilder/extractor/delta_lake_metadata_extractor.py

```diff
@@ -311,6 +333,53 @@ def fetch_columns(self, schema: str, table: str) -> List[ScrapedColumnMetadata]:
                     parsed_columns[row['col_name']].set_is_partition(True)
         return list(parsed_columns.values())
 
+    def _iterate_nested_columns(self,
+                                parent: str,
+                                curr_field: Any,
```

Comment:
could we use `union ` for all three complex type instead any? I have been having a hard time figure out the field value definition...

Reply:
Done ✅